### PR TITLE
Minor Performances Improvement:

### PR DESCRIPTION
**Pull request by arturobernalg**
* Unnecessary enum modifier
* Unnecessary interface modifier
* Unnecessary conversion to String
* Unnecessary semicolon
* Unused import

### CODE REVIEW

File: mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/tcp/MultiThreadedSocketServer.java [hunk 1]
```diff
@@ -262,7 +261,6 @@ public void run() {
           }
           if (clientCommand != null) {
             if (clientCommand instanceof Integer) {
-              ;
             } else {
```
Comment (TanguySoto) [inline mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/tcp/MultiThreadedSocketServer.java:264]: I'm not big fan of empty if() without even a comment. Could you please refactor those lines at the same time?

File: mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/threading/CentralNode.java [hunk 2]
```diff
@@ -91,7 +91,6 @@ void dataIn(Object obj) {
     if (obj instanceof CelestiaData && this.celestiaInterfaceServer != null) {
       this.celestiaInterfaceServer.putDataInBuffer(obj);
     } else if (this.multiThreadedSocketServer == null) {
-      ;
     } else {
```
Comment (TanguySoto) [inline mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/threading/CentralNode.java:94]: Same remark as above

File: mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/util/wav/WavFile.java [hunk 3]
```diff
@@ -40,8 +40,9 @@
 
 public class WavFile
 {
-	private enum IOState {READING, WRITING, CLOSED};
-	private final static int BUFFER_SIZE = 4096;
+	private enum IOState {READING, WRITING, CLOSED}
+
+    private final static int BUFFER_SIZE = 4096;
```
Comment (TanguySoto) [inline mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/util/wav/WavFile.java:45]: Bad formatting.

**Review by TanguySoto — CHANGES_REQUESTED** [review]
Nothing crazy, just a few comments 👍

File: mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/tcp/MultiThreadedSocketServer.java (hunk 1 above)
Comment (arturobernalg) [reply mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/tcp/MultiThreadedSocketServer.java]: Me neither. Changed

File: mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/util/wav/WavFile.java (hunk 3 above)
Comment (arturobernalg) [reply mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/util/wav/WavFile.java]: mmm Don't know what happened here. My mistake.

File: mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/threading/CentralNode.java (hunk 2 above)
Comment (arturobernalg) [reply mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/threading/CentralNode.java]: Changed

**Comment by dmarszk** [comment]
Hi @arturobernalg - sorry for delayed processing of these. I started merging them but then noticed they are targeting master branch - can you rebase them no top of the dev ?

**Comment by arturobernalg** [comment]
> Hi @arturobernalg - sorry for delayed processing of these. I started merging them but then noticed they are targeting master branch - can you rebase them no top of the dev ?

Done
TY

**Pull request merged**